cmd/skopeo/utils.go

```diff
@@ -108,6 +117,19 @@ func imageFlags(global *globalOptions, shared *sharedImageOptions, flagPrefix, c
 	return fs, opts
 }
 
+type retryOptions struct {
+	maxRetry   int           // The number of times to retry
+	retryDelay time.Duration // Time to wait when only retry once
+}
+
+func retryFlags() (pflag.FlagSet, *retryOptions) {
+	opts := retryOptions{}
+	fs := pflag.FlagSet{}
+	fs.IntVar(&opts.maxRetry, "retry-times", 0, "the number of times to retry, retry wait time will be exponential backoff delay based on the time of failed attempts")
```

Comment:
Maybe just
```suggestion
	fs.IntVar(&opts.maxRetry, "retry-times", 0, "the number of times to possibly retry")
```
? We can talk about the time in the description of the second option, and maybe say more about the kinds of errors that cause a retry in the man page.